src/recordlinker/routes/seed_router.py

```diff
@@ -0,0 +1,92 @@
+"""
+recordlinker.routes.seed_router
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the seed router for the RecordLinker API. Exposing
+the seed API endpoints.
+"""
+
+import typing
+
+import fastapi
+import sqlalchemy.orm as orm
+
+from recordlinker import models
+from recordlinker import schemas
+from recordlinker.database import get_session
+from recordlinker.linking import mpi_service as service
+
+router = fastapi.APIRouter()
+
+
+@router.post("", summary="Batch seed records", status_code=fastapi.status.HTTP_201_CREATED)
+def batch(
+    data: schemas.ClusterGroup, session: orm.Session = fastapi.Depends(get_session)
+) -> schemas.PersonGroup:
```

Comment:
Thikning of using this endpoint to seed the database with a list of PIIRecords. Would I need to structure each `PIIRecord` in its own cluster when sending the request?

Reply:
Yeah that will be easy enough to wrap the patients with a cluster and then a cluster group with all of them. Thanks!